WordPress/Classes/Extensions/Notifications/NotificationBlock+Interface.swift

```diff
@@ -3,53 +3,72 @@ import Foundation
 
 extension NotificationBlock
 {
-    public func subjectAttributedText() -> NSAttributedString {
-        let richSubjectCacheKey = "richSubject"
-        
-        if let cachedSubject = cacheValueForKey(richSubjectCacheKey) as? NSAttributedString {
-            return cachedSubject
+    // MARK: - Text Formatting Helpers
+    //
+    public func attributedSubjectText() -> NSAttributedString {
```

Comment:
Hey @jleandroperez, I'm still reviewing this, but one thing that could make things a bit easier is to document the class and its functions - not to explain what they're doing which should be figured out by the reviewer, but rather what they're supposed to do (ie: their purpose).